src/controller_examples/zookeeper_controller/trusted/spec_types.rs

```diff
@@ -7,7 +7,7 @@ use crate::kubernetes_api_objects::{
 };
 use crate::kubernetes_cluster::spec::{cluster::*, cluster_state_machine::*, message::*};
 use crate::vstd_ext::string_view::*;
-use crate::zookeeper_controller::{common::*, spec::zookeeper_api::*};
+use crate::zookeeper_controller::trusted::{step::*, zookeeper_api_model::*};
```

Comment:
Maybe I missed anything, but does this file need to import zookeeper_api_model?

Reply:
```rust
pub type ZKStep = Step<ZKMessage>;

pub type ZKCluster = Cluster<ZookeeperClusterView, ZKAPI, ZookeeperReconciler>;

pub type ZKMessage = Message<ZKAPIInputView, ZKAPIOutputView>;
```

I think the `ZKAPI` and related types are define there?

Reply:
oh I see